lightspeed/modules/con_lightspeed-process.adoc

```diff
@@ -16,14 +16,14 @@ To use {LightspeedFullName}, ensure that you have the following components:
 == Process for using Red Hat Ansible Lightspeed
 
 * Red Hat customer Portal organization administrators can use this guide to: 
-** xref:assembly_configure-code-assistant[Connect {LightspeedShortName} to {ibmwatsonxcodeassistant} instance]
-** xref:proc_obtain-config-wca-and-model-id[Configure the {ibmwatsonxcodeassistant} model that you want to use]
-** xref:assembly_assigning-seat-licenses[Assign and remove {LightspeedShortName} seat licenses]
+** xref:configure-code-assistant_lightspeed-user-guide[Connect {LightspeedShortName} to {ibmwatsonxcodeassistant} instance]
+** xref:obtain-config-wca-and-model-id_configure-code-assistant[Configure the {ibmwatsonxcodeassistant} model that you want to use]
+** xref:assigning-seat-licenses_lightspeed-user-guide[Assign and remove {LightspeedShortName} seat licenses]
 * Ansible users (content developers and automators) can use this guide to: 
-** xref:assembly_configuring-vscode-extension[Install and configure Ansible Visual Studio Code extension]
-** xref:assembly_requesting-task-recommendations[Request task recommendations]
-** xref:proc_provide-feedback[Provide feedback on the Ansible Lightspeed service]
+** xref:configuring-with-code-assistant_lightspeed-user-guide[Install and configure Ansible Visual Studio Code extension]
```

Comment:
```suggestion
** xref:configuring-with-code-assistant_lightspeed-user-guide[Install and configure Ansible Visual Studio Code extension]
```
I linked to an assembly that has the title "Installing and configuring the Ansible VScode extension" but this assembly has context set to  `configuring-with-code-assistant` 

There's also a module with a similar name, so I didn't know which to use.